models/yang/sonic/sonic-ifa.yang

```diff
@@ -0,0 +1,77 @@
+module sonic-ifa {
+    namespace "http://github.com/Azure/sonic-ifa";
+    prefix ifa;
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC TAM IFA";
+
+    revision 2019-11-12 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-ifa {
+
+        container TAM_INT_IFA_FEATURE_TABLE {
+
+            list TAM_INT_IFA_FEATURE_TABLE_LIST {
+                key "name";
+
+                leaf name {
+                    type enumeration {
+                        enum feature;
```

Comment:
Is this compiling fine ? Note that "feature" is reserved keyword.

Reply:
We may use type string {
				pattern 'feature';
			}

Reply:
I currently have it defied as below:
leaf name {
type enumeration {
enum feature;
}
}

This compiles and works just fine. The pattern and enumeration AFAIK are the same.